ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileClustersTable.tsx

```diff
@@ -66,127 +66,135 @@ function ProfileClustersTable({
+                            <Th modifier="fitContent" width={10}>
+                                Compliance
+                            </Th>
+                        </Tr>
+                    </Thead>
+                    <TbodyUnified
+                        tableState={tableState}
+                        colSpan={7}
+                        errorProps={{
+                            title: 'There was an error loading profile clusters',
+                        }}
+                        emptyProps={{
+                            message:
+                                'If you have recently created a scan schedule, please wait a few minutes for the results to become available.',
+                        }}
+                        filteredEmptyProps={{ onClearFilters }}
+                        renderer={({ data }) => (
+                            <Tbody>
+                                {data.map((clusterInfo) => {
+                                    const {
+                                        cluster: { clusterId, clusterName },
+                                        lastScanTime,
+                                        checkStats,
+                                    } = clusterInfo;
+                                    const {
+                                        passCount,
+                                        failCount,
+                                        manualCount,
+                                        otherCount,
+                                        totalCount,
+                                    } = getStatusCounts(checkStats);
+                                    const passPercentage = calculateCompliancePercentage(
+                                        passCount,
+                                        totalCount
+                                    );
+                                    const progressBarId = `progress-bar-${clusterId}`;
+                                    const firstDiscoveredAsPhrase = getDistanceStrictAsPhrase(
+                                        lastScanTime,
+                                        currentDatetime
+                                    );
 
-                                return (
-                                    <Tr key={clusterId}>
-                                        <Td dataLabel="Cluster">
-                                            <Link
-                                                to={generatePathWithScanConfig(
-                                                    coverageClusterDetailsPath,
-                                                    {
-                                                        clusterId,
-                                                        profileName,
+                                    return (
+                                        <Tr key={clusterId}>
+                                            <Td dataLabel="Cluster" modifier="fitContent">
```

Comment:
It looks like `<Td dataLabel="Check">` in the other table is only that missed `modifier="fitContent"` prop.

Reply:
I went with a `min-width` on that one instead because the check name can be very long, so I figured wrapping on that one would be okay? thoughts?

Here's an example check "rhcos4-stig-v1r1-master-auditd-data-retention-max-log-file-action-stig"

Reply:
Here's what that would look like on 1440px (default 15in mac effective resolution)

With wrapping:
![Screenshot 2024-07-01 at 10 41 17 AM](https://github.com/stackrox/stackrox/assets/61400697/0e02d10f-b590-4b1c-9821-3644456ea9c9)

Without wrapping:
![Screenshot 2024-07-01 at 10 41 08 AM](https://github.com/stackrox/stackrox/assets/61400697/a39a0a8d-296a-4bc5-bf62-5baf0a7959f9)